.circleci/readme.md

```diff
@@ -58,8 +58,8 @@ dtest repo and branch with MIDRES config you can run:
 
 ```
 generate.sh -m \
-  -e DTEST_REPO=git://github.com/adelapena/cassandra-dtest.git \
-  -e DTEST_BRANCH=CASSANDRA-8272 
+  -e DTEST_REPO=git://github.com/ekaterinadimitrova2/cassandra-dtest.git \
+  -e DTEST_BRANCH=CASSANDRA-15234-new
```

Comment:
nit: I'm guessing this isn't strictly necessary...maybe the result of a find-replace?

Reply:
Part of circle trash to be reverted